notebooks/official/pipelines/custom_model_training_and_batch_prediction.ipynb

```diff
@@ -2,13 +2,13 @@
   "cells": [
```

Comment:
<p>missing deleting the pipeline job</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/835/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>